Cauldron/Assets/Scripts/IngredientDroppable.cs

```diff
@@ -160,11 +160,19 @@ private void OverCauldron()
 
         private void AddToMixAndResetVisuals()
         {
-            cauldron.AddToMix(ingredient);
             dragging = false;
             dragTrail?.SetActive(false);
             transform.position = initialPosition;
             transform.DOScale(transform.localScale, rotateSpeed).From(Vector3.zero);
+
+            try
+            {
+                cauldron.AddToMix(ingredient);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e.Message);
```

Comment:
It's usually better to write something like "Error while adding ingredient to mix: {message}" when handling exceptions.